playbooks/ansible/roles/ctdb.setup/tasks/cephfs/main.yml

```diff
@@ -6,4 +6,3 @@
   vars:
     name: "ctdb"
     path: "{{ ctdb_lock_path }}"
-    mode: "0777"
```

Comment:
So the default _0755_ is enough I guess.

Reply:
Yes. CTDB runs as root, so there's no need to grant write access permissions to other users.